### PR TITLE
Add Velocity to downloads page + separate builds into "promoted" or not

### PR DESCRIPTION
**Pull request by Aberdeener**
image

image

image

### CODE REVIEW

File: src/js/downloads.js [hunk 1]
```diff
@@ -104,25 +118,54 @@ function load(id) {
...
+                            </a>
+                        </td>
+                     </tr>`;
+
+        build.promoted
+            ? promotedRows += row
+            : rows += row;
     });
 
+    const noBuilds = `<tr style="background-color: #181A1B">
```
Comment (NotMyFault) [inline src/js/downloads.js:152]: Using a static color here leads to it looking out of line when using a different theme than darkmode. Instead, work with `prefers-color-scheme` and switch colors depending on what the user requests.

**Comment by MiniDigger** [comment]
this supersedes #122, right?

**Comment by NotMyFault** [comment]
> this supersedes #122, right?

Yes, aber got it covered.

File: src/js/downloads.js (hunk 1 above)
Comment (Aberdeener) [reply src/js/downloads.js]: Great point, fixed!

**Review by kashike — APPROVED** [review]

**Pull request merged**